components/ui-shell/settings.ts

```diff
@@ -186,8 +186,8 @@ export const NAV_ITEMS: TopLevelNavItem[] = [
         label: 'Help and Learning',
         children: [
           {
-            label: 'Stack Exchange',
-            url: 'https://quantumcomputing.stackexchange.com/questions/tagged/qiskit',
+            label: 'Support Channels',
+            url: 'https://qisk.it/support',
```

Comment:
Support channels page contain information about Slack as well. Should we remove the Slack link in this drop down menu?

Reply:
I noticed that too. I think it depends how much we want people going to Slack? For Pantsbuild, that was a huge project priority so we referenced Slack _everywhere_ like blog posts. Even if it was repetitive like this here would be.

For Qiskit, how much do want to specifically funnel people to Slack? If we want to, then I think it's worth repeating. Otherwise, we could probably remove.

The benefit of removing would be less text in the top nav bar, I suppose?

Reply:
For me the benefits of removing it is so that people will read the information about different slack channels and where to get help for specific issues, similar to the spirit of this PR trying to guide people to read about what kind of questions should go to StackExchange instead of sending people there directly.